cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -196,6 +197,12 @@ public MasterEnvironmentRunnable createRunnable(MasterEnvironmentContext context
     return cls.getConstructor(MasterEnvironmentContext.class, MasterEnvironment.class).newInstance(context, this);
   }
 
+  @Override
+  public MasterEnvironmentRunnable createRunnable(MasterEnvironmentRunnableContext context,
```

Comment:
Why there are two methods for creating the Runnable? There should only be one.

Reply:
createRunnable(MasterEnvironmentRunnableContext context.....) in MasterEnv is part of SPI. I suppose we cannot remove the existing one.  Marked it as deprecated 

  @Deprecated
  MasterEnvironmentRunnable createRunnable(MasterEnvironmentContext context,
                                           Class<? extends MasterEnvironmentRunnable> runnableClass) throws Exception;

Reply:
We can just remove it. The SPI is very new and we should try to keep it clean for now

Reply:
Done. Removed